src/gtest/test_libzendoo.cpp

```diff
@@ -58,61 +61,82 @@ TEST(ZendooLib, PoseidonHashTest)
     auto rhs_field = zendoo_deserialize_field(rhs);
     ASSERT_TRUE(rhs_field != NULL);
 
+
     auto expected_hash = zendoo_deserialize_field(hash);
     ASSERT_TRUE(expected_hash != NULL);
 
-    const field_t* hash_input[] = {lhs_field, rhs_field};
+    auto digest = ZendooPoseidonHash(NULL, 0);
```

Comment:
This constructor is a little bit ugly when no personalization is needed. I will add an empty one too in zendoo-mc-cryptolib.